src/parser/parser.py

```diff
@@ -233,9 +239,13 @@ def parse_declaration(self, ident = None):
             TokenType.SAN
         ]
         if not self.expect_peek_in(data_types):
-            self.no_data_type_error(self.peek_tok)
-            self.advance(2)
-            return None
+            # Check for class id as data type (check Token's token str)
+            if self.peek_tok.token.token[:6] == "CWASS_":
```

Comment:
Use `.startswith()` instead of indexing

Reply:
also, consider turning this into a method. See `expect_peek_is_identifier()` method for more info

Reply:
okokok thank you